src/V3Force.cpp

```diff
@@ -64,15 +64,6 @@ class ForceConvertVisitor final : public VNVisitor {
             m_rdVarp->addNext(m_enVarp);
             m_rdVarp->addNext(m_valVarp);
             varp->addNextHere(m_rdVarp);
-
-            if (varp->isPrimaryIO()) {
-                varp->v3warn(
-                    E_UNSUPPORTED,
-                    "Unsupported: Force/Release on primary input/output net "
-                        << varp->prettyNameQ() << "\n"
-                        << varp->warnMore()
-                        << "... Suggest assign it to/from a temporary net and force/release that");
-            }
```

Comment:
t_force_mid tries it on a primary output, but please add a test to also check it works as a primary output.

Also t_force_mid.out should be deleted.

With those fixed and tests passing feel free to merge

Reply:
test it on a primary **input**.